index.js

```diff
@@ -226,33 +307,96 @@ module.exports = {
     // when setting default values or when transforming a parameter that can be
     // supplied in multiple formats into a canonical one.
     "no-param-reassign": "off",
+    "no-plusplus": "off", // I don't want this joyless world.
     "no-proto": "error",
     "no-redeclare": "error",
+
+    // Disabled because when matching an exact number of spaces, writing them
+    // out is usually better than using the `{n}` syntax. It makes the regexp
+    // more similar to matched strings.
+    "no-regex-spaces": "off",
+    "no-restricted-exports": "off",
+    "no-restricted-globals": "off",
+    "no-restricted-imports": "off",
     "no-restricted-properties": "off",
+    "no-restricted-syntax": "off",
     "no-return-assign": "error",
     "no-return-await": "error",
     "no-script-url": "error",
-    "no-self-assign": "error",
-    "no-self-compare": "error",
     "no-sequences": "error",
+    "no-shadow": "error",
```

Comment:
This is going to cause good changes in Peggy.

Reply:
I'm not sure... It creates a lot of strange renames (when function parameter shadows external variable) without much benefit, IMO. But I'm not strongly against it. Maybe enable shadowing for some names (`node`, `context`)?

Reply:
I'll turn it off for now.  Perhaps we can enable it some other time.